fix: Match on the whole markdown tag when finding existing comments

tim775 opened the pull request:
Fixes https://github.com/infracost/actions/issues/60

vdmgolub commented on internal/comment/bitbucket.go:
```diff
@@ -794,3 +794,7 @@ func (h *bitbucketServerPRHandler) fetchServerComment(commentURL string) (*bitbu
 }
+
+func bitbucketMarkdownTag(tag string) string {
+	return fmt.Sprintf("\n\n*(%s)*", tag)
```
I wonder if we should include these two empty lines, it's more formatting to me, `*(infracost-comment)*` should be enough to detect. What do you think?

tim775 commented:
[//]: <> (infracost-comment)

💰 Infracost estimate: **monthly cost will increase by ₹10,089 (+58%) 📈**
<table>
  <thead>
    <td>Project</td>
    <td>Previous</td>
    <td>New</td>
    <td>Diff</td>
  </thead>
  <tbody>
    <tr>
      <td>infracost/infra/dev</td>
      <td align="right">₹17,269</td>
      <td align="right">₹27,358</td>
      <td>+₹10,089 (+58%)</td>
    </tr>
  </tbody>
</table>

<details>
<summary><strong>Infracost output</strong></summary>

```
Project: infracost/infra/dev

~ module.base.module.eks.module.node_groups.aws_eks_node_group.workers["default"]
  +₹10,089 (₹5,578 → ₹15,667)

    ~ Instance usage (Linux/UNIX, on-demand, t3.small)
      -₹3,363 (₹3,363 → ₹13,452)

    ~ Instance usage (Linux/UNIX, on-demand, t3.large)
      +₹13,452 (₹3,363 → ₹13,452)

Monthly cost change for infracost/infra/dev
Amount:  +₹10,089 (INR) (₹17,269 → ₹27,358)
Percent: +58%

──────────────────────────────────
Key: ~ changed, + added, - removed

```
</details>

This comment will be updated when the cost estimate changes.

<sub>
  Is this comment useful? <a href="https://www.infracost.io/feedback/submit/?value=yes" rel="noopener noreferrer" target="_blank">Yes</a>, <a href="https://www.infracost.io/feedback/submit/?value=no" rel="noopener noreferrer" target="_blank">No</a>
</sub>

tim775 commented:
[//]: <> (prod)

💰 Infracost estimate: **monthly cost will increase by ₹10,089 (+58%) 📈**
<table>
  <thead>
    <td>Project</td>
    <td>Previous</td>
    <td>New</td>
    <td>Diff</td>
  </thead>
  <tbody>
    <tr>
      <td>infracost/infra/dev</td>
      <td align="right">₹17,269</td>
      <td align="right">₹27,358</td>
      <td>+₹10,089 (+58%)</td>
    </tr>
  </tbody>
</table>

<details>
<summary><strong>Infracost output</strong></summary>

```
Project: infracost/infra/dev

~ module.base.module.eks.module.node_groups.aws_eks_node_group.workers["default"]
  +₹10,089 (₹5,578 → ₹15,667)

    ~ Instance usage (Linux/UNIX, on-demand, t3.small)
      -₹3,363 (₹3,363 → ₹13,452)

    ~ Instance usage (Linux/UNIX, on-demand, t3.large)
      +₹13,452 (₹3,363 → ₹13,452)

Monthly cost change for infracost/infra/dev
Amount:  +₹10,089 (INR) (₹17,269 → ₹27,358)
Percent: +58%

──────────────────────────────────
Key: ~ changed, + added, - removed

```
</details>

This comment will be updated when the cost estimate changes.

<sub>
  Is this comment useful? <a href="https://www.infracost.io/feedback/submit/?value=yes" rel="noopener noreferrer" target="_blank">Yes</a>, <a href="https://www.infracost.io/feedback/submit/?value=no" rel="noopener noreferrer" target="_blank">No</a>
</sub>

tim775 commented:
[//]: <> (prod)

💰 Infracost estimate: **monthly cost will increase by ₹10,089 (+58%) 📈**
<table>
  <thead>
    <td>Project</td>
    <td>Previous</td>
    <td>New</td>
    <td>Diff</td>
  </thead>
  <tbody>
    <tr>
      <td>infracost/infra/dev</td>
      <td align="right">₹17,269</td>
      <td align="right">₹27,358</td>
      <td>+₹10,089 (+58%)</td>
    </tr>
  </tbody>
</table>

<details>
<summary><strong>Infracost output</strong></summary>

```
Project: infracost/infra/dev

~ module.base.module.eks.module.node_groups.aws_eks_node_group.workers["default"]
  +₹10,089 (₹5,578 → ₹15,667)

    ~ Instance usage (Linux/UNIX, on-demand, t3.small)
      -₹3,363 (₹3,363 → ₹13,452)

    ~ Instance usage (Linux/UNIX, on-demand, t3.large)
      +₹13,452 (₹3,363 → ₹13,452)

Monthly cost change for infracost/infra/dev
Amount:  +₹10,089 (INR) (₹17,269 → ₹27,358)
Percent: +58%

──────────────────────────────────
Key: ~ changed, + added, - removed

```
</details>

This comment will be updated when the cost estimate changes.

<sub>
  Is this comment useful? <a href="https://www.infracost.io/feedback/submit/?value=yes" rel="noopener noreferrer" target="_blank">Yes</a>, <a href="https://www.infracost.io/feedback/submit/?value=no" rel="noopener noreferrer" target="_blank">No</a>
</sub>

tim775 commented:
[//]: <> (prod)

💰 Infracost estimate: **monthly cost will not change**
<table>
  <thead>
    <td>Project</td>
    <td>Previous</td>
    <td>New</td>
    <td>Diff</td>
  </thead>
  <tbody>
    <tr>
      <td>infracost/infra/dev</td>
      <td align="right">₹17,269</td>
      <td align="right">₹17,269</td>
      <td>₹0</td>
    </tr>
  </tbody>
</table>

<details>
<summary><strong>Infracost output</strong></summary>

```
──────────────────────────────────

The following projects have no cost estimate changes: infracost/infra/dev
Run infracost breakdown to see their breakdown.

──────────────────────────────────

```
</details>

This comment will be updated when the cost estimate changes.

<sub>
  Is this comment useful? <a href="https://www.infracost.io/feedback/submit/?value=yes" rel="noopener noreferrer" target="_blank">Yes</a>, <a href="https://www.infracost.io/feedback/submit/?value=no" rel="noopener noreferrer" target="_blank">No</a>
</sub>

tim775 commented:
[//]: <> (infracost-comment)

💰 Infracost estimate: **monthly cost will increase by ₹10,089 (+58%) 📈**
<table>
  <thead>
    <td>Project</td>
    <td>Previous</td>
    <td>New</td>
    <td>Diff</td>
  </thead>
  <tbody>
    <tr>
      <td>infracost/infra/dev</td>
      <td align="right">₹17,269</td>
      <td align="right">₹27,358</td>
      <td>+₹10,089 (+58%)</td>
    </tr>
  </tbody>
</table>

<details>
<summary><strong>Infracost output</strong></summary>

```
Project: infracost/infra/dev

~ module.base.module.eks.module.node_groups.aws_eks_node_group.workers["default"]
  +₹10,089 (₹5,578 → ₹15,667)

    ~ Instance usage (Linux/UNIX, on-demand, t3.small)
      -₹3,363 (₹3,363 → ₹13,452)

    ~ Instance usage (Linux/UNIX, on-demand, t3.large)
      +₹13,452 (₹3,363 → ₹13,452)

Monthly cost change for infracost/infra/dev
Amount:  +₹10,089 (INR) (₹17,269 → ₹27,358)
Percent: +58%

──────────────────────────────────
Key: ~ changed, + added, - removed

```
</details>

This comment will be updated when the cost estimate changes.

<sub>
  Is this comment useful? <a href="https://www.infracost.io/feedback/submit/?value=yes" rel="noopener noreferrer" target="_blank">Yes</a>, <a href="https://www.infracost.io/feedback/submit/?value=no" rel="noopener noreferrer" target="_blank">No</a>
</sub>

tim775 commented:
[//]: <> (infracost-comment)

💰 Infracost estimate: **monthly cost will increase by ₹10,089 (+58%) 📈**
<table>
  <thead>
    <td>Project</td>
    <td>Previous</td>
    <td>New</td>
    <td>Diff</td>
  </thead>
  <tbody>
    <tr>
      <td>infracost/infra/dev</td>
      <td align="right">₹17,269</td>
      <td align="right">₹27,358</td>
      <td>+₹10,089 (+58%)</td>
    </tr>
  </tbody>
</table>

<details>
<summary><strong>Infracost output</strong></summary>

```
Project: infracost/infra/dev

~ module.base.module.eks.module.node_groups.aws_eks_node_group.workers["default"]
  +₹10,089 (₹5,578 → ₹15,667)

    ~ Instance usage (Linux/UNIX, on-demand, t3.small)
      -₹3,363 (₹3,363 → ₹13,452)

    ~ Instance usage (Linux/UNIX, on-demand, t3.large)
      +₹13,452 (₹3,363 → ₹13,452)

Monthly cost change for infracost/infra/dev
Amount:  +₹10,089 (INR) (₹17,269 → ₹27,358)
Percent: +58%

──────────────────────────────────
Key: ~ changed, + added, - removed

```
</details>

This comment will be updated when the cost estimate changes.

<sub>
  Is this comment useful? <a href="https://www.infracost.io/feedback/submit/?value=yes" rel="noopener noreferrer" target="_blank">Yes</a>, <a href="https://www.infracost.io/feedback/submit/?value=no" rel="noopener noreferrer" target="_blank">No</a>
</sub>
